docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -220,12 +259,64 @@ public interface RestClientWithTokenPropagationFilter {
     Uni<String> getAdminName();
 }
 ----
+<1> Register an OIDC token propagation filter with the REST client to propagate the incoming already-existing tokens.
+
+Note that `RestClientWithOidcClientFilter` and `RestClientWithTokenPropagationFilter` interfaces are the same.
+This is because combining OIDC client and token propagation filters on the same REST Client causes side effects because both filters can interfere with each other.
+For example, the OIDC client filter can override the token propagated by the OIDC token propagation filter, or the OIDC token propagation filter can fail if it is called when no token is available to propagate and the OIDC client filter is expected to get a new token instead.
```

Comment:
Completely optional suggestion:
```suggestion
IMPORTANT: Do not use the `RestClientWithOidcClientFilter` and `RestClientWithTokenPropagationFilter` interfaces in the same REST client because they can conflict, leading to issues.
For example, the OIDC client filter can override the token from the OIDC token propagation filter, or the propagation filter might not work correctly if it attempts to propagate a token when none is available, expecting the OIDC client filter to obtain a new token instead.
```